src/lib.rs

```diff
@@ -37,7 +110,27 @@ pub fn load<T: Serialize + DeserializeOwned + Default>(name: &str) -> Result<T,
     }
 }
 
-/// Store a configuration object
+/// Save changes made to a configuration object
+///
+/// This function does not assume that a configuration
+/// already exists. It will create a new one, with the
+/// provided values, if none exists.
+///
+/// You can also use this function to create a new configuration
+/// with different initial values than which are provided
+/// by your `Default` trait implementation, or if your
+/// configuration structure _can't_ implement `Default`.
+///
+/// ```rust,ignore
+///     let my_cfg = MyConf { ... };
+///     confy::store(my_cfg)?;
+/// }
```

Comment:
identation and that weird brace here

can you make this `no_run`?

Reply:
Same as above :P